code/game/objects/machinery/camera/camera.dm

```diff
@@ -310,10 +309,21 @@
 	user.see_in_dark = 2
 	return TRUE
 
-
 /obj/machinery/camera/autoname
+	light_range = 1
+	light_power = 0.2
```

Comment:
Does this need adding?

Reply:
Yes, emissives won't glow in the dark if there isn't a minimum amount of light level.
When light is at 0, humans can't see it (no night vision), which defeats the purpose.
I wanted the green bits of the camera eye to be visible in darkness.